test/tests/functional/pbs_mom_mock_run.py

```diff
@@ -68,5 +68,6 @@ def test_rsc_used(self):
         self.server.accounting_match(msg=used_ncpus, id=jid)
         used_mem = "resources_used.mem=5mb"
         self.server.accounting_match(msg=used_mem, id=jid)
-        used_walltime = "resources_used.walltime=00:00:05"
-        self.server.accounting_match(msg=used_walltime, id=jid)
+        used_walltime = "resources_used.walltime=00:00:00"
+        self.server.accounting_match(
+            msg=used_walltime, existence=False, id=jid)
```

Comment:
doing existence=False without a max_attempts will take 30s.  I see two ways to go about this.  Just add a max_attempts=10 or something is one.  The other way is a little smarter,  You can add an existence match of just 'resources_used.walltime', and then do a existence=False, max_attempts=1 match for what you are looking for.